src/daft-dsl/src/resolve_expr.rs

```diff
@@ -348,6 +375,22 @@ pub fn resolve_aggexprs(
     Ok((resolved_exprs, resolved_fields?))
 }
 
+pub fn is_valid_column_name(name: &str, schema: &Schema) -> bool {
```

Comment:
WDYT of raising an exception here instead? That would have a lot more context around the failure rather than just true/false (e.g. "name has no wildcard matches")

Reply:
Sounds good